specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/ConnectToSourceSqlServerTask.json

```diff
@@ -0,0 +1,261 @@
+          },
+          "readOnly": true
+        },
+        "compatibilityLevel": {
+          "$ref": "./TasksCommon.json#/definitions/DatabaseCompatLevel",
+          "description": "SQL Server compatibility level of database",
+          "readOnly": true
+        },
+        "databaseState": {
+          "$ref": "./TasksCommon.json#/definitions/DatabaseState",
+          "description": "State of the database",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ConnectToSourceSqlServerTaskOutput"
+        }
+      ]
+    },
+    "ConnectToSourceSqlServerTaskOutputLoginLevel": {
+      "x-ms-discriminator-value": "LoginLevelOutput",
+      "type": "object",
+      "description": "Login level output for the task that validates connection to SQL Server and also validates source server requirements",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Login name.",
+          "readOnly": true
+        },
+        "loginType": {
+          "$ref": "./TasksCommon.json#/definitions/LoginType",
+          "description": "The type of login.",
+          "readOnly": true
+        },
+        "defaultDatabase": {
+          "type": "string",
+          "description": "The default database for the login.",
+          "readOnly": true
+        },
+        "isEnabled": {
+          "type": "boolean",
+          "description": "The state of the login.",
+          "readOnly": true
+        },
+        "migrationEligibility": {
+          "$ref": "./TasksCommon.json#/definitions/MigrationEligibilityInfo",
+          "description": "Information about eligiblity of login for migration.",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ConnectToSourceSqlServerTaskOutput"
+        }
+      ]
+    },
+    "ConnectToSourceSqlServerTaskOutputAgentJobLevel": {
+      "x-ms-discriminator-value": "AgentJobLevelOutput",
+      "type": "object",
+      "description": "AgentJob level output for the task that validates connection to SQL Server and also validates source server requirements",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "AgentJob name",
+          "readOnly": true
+        },
+        "jobCategory": {
+          "type": "string",
+          "description": "The type of AgentJob.",
+          "readOnly": true
+        },
+        "isEnabled": {
+          "type": "boolean",
+          "description": "The state of the original AgentJob.",
+          "readOnly": true
+        },
+        "jobOwner": {
+          "type": "string",
+          "description": "The owner of the AgentJob",
+          "readOnly": true
+        },
+        "lastExecutedOn": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "UTC Date and time when the AgentJob was last executed."
```

Comment:
Minor - Can you replace 'AgentJob' in descriptions with 'Agent Job'. Thanks!

Reply:
Done.